docs/source/guide/install.md

```diff
@@ -117,21 +118,49 @@ conda activate label-studio
 pip install label-studio
 ```
 
+## Install on Google Cloud
+
+1. First login with `gcloud`
+```bash
+gcloud auth application-default login
+```
+
+2. You can use `direnv` to keep track of necessary env variables. Set up `.envrc` and don't forget to run `direnv allow`.
+```bash
+export PROJECT_ID=maystra-poc
+```
+
+3. Build the docker image and push it to Google Build with:
```

Comment:
Does this command also build the Docker image? If no, please make these 2 separate steps:
```suggestion
3. Build the Docker image.
4. Push the Docker image you built to Google Build using the following command:
```
If this command does build the docker image, then minor suggestions:
```suggestion
3. Build the Docker image and push it to Google Build using the following command:
```